### ISSUE TITLE
OOM / huge image files

### ISSUE THREAD

**Issue by Bapho** [coomar2841/image-chooser-library#109]
I'm facing a OOM Error in the MediaProcessorThread when choosing huge image files.
The options.inSampleSize variable should be calculated relatively to the "container size".
Please check http://stackoverflow.com/a/9234895/3410272

**Comment by coomar2841** [comment]
Yep, I am planning to include that in the next release. Caught up with some work. Can't find enough time right now. Perhaps, in a week or two, a new release should happen.

**Comment by Bapho** [comment]
Great!
Will png's be saved as png's? this possibly could save some disk size.

**Comment by coomar2841** [comment]
Will take that as a different bug.
